Tests/Command/RemoveThumbsCommandTest.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Tests\Command;
+
+use Sonata\MediaBundle\Command\RemoveThumbsCommand;
+use Sonata\MediaBundle\Model\MediaManagerInterface;
+use Sonata\MediaBundle\Provider\Pool;
+use Sonata\MediaBundle\Tests\Entity\Media;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * @author Anton Dyshkant <vyshkant@gmail.com>
+ *
+ * @requires function Symfony\Component\Filesystem\Tests\FilesystemTestCase::setUpBeforeClass
+ * @requires function Symfony\Component\Console\Input::setStream
+ */
+class RemoveThumbsCommandTest extends TestCase
```

Comment:
Extend directly from FileSystemTestCase and remove `@requires function Symfony\Component\Filesystem\Tests\FilesystemTestCase::setUpBeforeClass` section above